general_rules/Scenario.tex

```diff
@@ -56,6 +56,9 @@ \subsection{Furniture}
 	\item a couch,
 	\item a small table,
 	\item a small dinner table with two chairs,
+	\item a trash bin,
+	\item a recycling bin,
+	\item a compost bin,
```

Comment:
This is a No-Go from my perspective. We are requesting two relatively big and expensive objects per arena to the LOC for a single, non-mandatory test. I'd just drop them and add only the trash bin. Maybe two small trash bins are OK.

The robot can take the kitchen and bathroom trash outside, to the appropriate designated location.

Reply:
Agree. Moved to a minimum of a single bin which may be small (though a second reduces the shuffling that the ref has to do).